### ISSUE TITLE
Training based on Conan 2.0 and windows based 

### ISSUE THREAD

**Issue by sangramjadhavbhtc** [conan-io/training#113]
Hi Team,
       The contains of the current version of training material is nice. But please provides training based on windows and latest version of conan.

**Comment by memsharded** [comment]
Hi @sangramjadhavbhtc 

The video trainings take a huge effort to be produced, and we don't have the resources at the moment. Maybe at the end of the year we will try, but it is not guaranteed.

What we did is completely change the docs, providing a hands-on tutorial section, with examples, code and explanations very similar to what the video trainings are doing, just in written form. The recommendation to learn Conan in a similar fashion would be to follow it, executing the examples and reading it all: https://docs.conan.io/2/tutorial.html

**Comment by sangramjadhavbhtc** [comment]
Thats true it will take lots of efforts. But, from my point of view, for the new user, how to start with conan and brief of basic features or some training sessions(windows based) are expected. 

The https://docs.conan.io/2/tutorial.html documents are really helpful.

**Comment by sangramjadhavbhtc** [comment]
Ok
